mrp_account_bom_attribute_match/tests/test_mrp_account_bom_attribute_match.py

```diff
@@ -21,3 +23,58 @@ def test_bom_cost(self):
         sword_magenta.button_bom_cost()
         self.assertEqual(sword_cyan.standard_price, 1.00)
         self.assertEqual(sword_magenta.standard_price, 2.00)
+
+    def test_bom_cost_template_skip_and_ignore(self):
```

Comment:
I don't understand this test, there is not a single assert so nothing is being actually tested.